e2e-tests/.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+test-results
```

Comment:
Minor note: I usually prefer a top-level `.gitignore`. From looking through our other `.gitignores` they seem to have quite a bit of duplication between each other and compared to the top-level one (see `merlin-server/.gitignore` which contains a lot of items that are in the main `.gitignore` anyways).

Reply:
I actually typically prefer the opposite for monorepos. The idea is each package is standalone and ignores what it needs.